cmd/influxd/inspect/inspect.go

```diff
@@ -21,6 +22,7 @@ func NewCommand() *cobra.Command {
 		//NewBuildTSICommand(),
 		//NewCompactSeriesFileCommand(),
 		//NewExportBlocksCommand(),
```

Comment:
I think `export-blocks` might have been ye olde 2.x name for this command, I can use that instead of `export` if it's still relevant.